CMakeLists.txt

```diff
@@ -12,6 +12,11 @@
 cmake_minimum_required(VERSION 3.7)
 project(CycloneDDS-CXX VERSION 0.1.0 LANGUAGES C CXX)
 
+set(PROJECT_NAME_FULL "Eclipse Cyclone DDS ISO IEC C++ PSM")
+
+# Set some convenience variants of the project-name
+string(REPLACE " " "-" PROJECT_NAME_DASHED "${PROJECT_NAME_FULL}")
+
```

Comment:
Please set this in the CMakeCPack.cmake file. It's not used anywhere in the project, and I've just removed all those silly instances in the xtypes branch.

Reply:
i moved it to CMakeCPack.cmake